bundle-core/src/main/java/net/discdd/pathutils/TransportPaths.java

```diff
@@ -0,0 +1,36 @@
+package net.discdd.pathutils;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.SEVERE;
+
+public class TransportPaths{
+    private static final Logger logger = Logger.getLogger(TransportPaths.class.getName());
+
+    private final Path clientPath;
+    private final Path serverPath;
```

Comment:
make public. change to toClientPath and toServerPath to get the directions clear.